src/UI/ManualImport/Artist/SelectArtistLayout.js

```diff
@@ -18,30 +18,30 @@ module.exports = Marionette.Layout.extend({
 
     columns : [
         {
-            name      : 'title',
-            label     : 'Title',
+            name      : 'name',
```

Comment:
Relatively sure that label is just a string in the UI, but wanted to make sure the different capitalization here was intended. It looks like there's the same in a later file.

Reply:
Yes, name reads the variable, label is the UI column header